src/helpers/sync_helpers.sh

```diff
@@ -18,11 +18,11 @@ function sync_branch_steps {
       echo "merge_tracking_branch"
       echo "merge $(parent_branch "$branch")"
     else
-      echo "rebase_tracking_branch"
+      echo "$PULL_BRANCH_STRATEGY""_tracking_branch"
```

Comment:
We have used that syntax before [here](https://github.com/Originate/git-town/blob/master/src/helpers/helpers.sh#L50-L51)